docs/machine-learning/tutorials/image-classification-api-transfer-learning.md

```diff
@@ -0,0 +1,456 @@
+---
+title: 'Tutorial: Automated visual inspection using transfer learning'
+description: This tutorial illustrates how to use transfer learning to train a TensorFlow deep learning model in ML.NET using the image detection API to classify images of concrete surfaces as cracked or not cracked.
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 10/03/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a developer, I want to use ML.NET so that I can use transfer learning in an image classification scenario to classify images using a pretrained TensorFlow model and ML.NET's Image Classification API.
+---
+
+# Tutorial: Automated visual inspection using transfer learning with ML.NET's Image Classification API
+
+Learn to train a custom deep learning model using transfer learning and the ML.NET Image Classification API to classify images of concrete surfaces as cracked or uncracked.
+
+> [!NOTE]
+> The ML.NET Image Classification API is currently in preview.
+
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+>
+> - Understand the problem
+> - Learn about ML.NET Image Classification API
+> - Understand the pretrained model
+> - Use transfer learning to train a custom TensorFlow image classification model
+> - Classify images with the custom model
+
+## Image Classification Transfer Learning Sample Overview
+
+This sample is a .NET Core console application that classifies images using a pre-trained deep learning TensorFlow model. The code for this sample can be found on the [dotnet/machinelearning-samples repository](https://github.com/dotnet/machinelearning-samples) on GitHub.
+
+## Understand the problem
+
+Image classification is a computer vision problem. Image classification takes an image as input and categorizes it into a prescribed class. Some scenarios where image classification is useful include:
+
+- Facial Recognition
+- Emotion detection
+- Medical diagnosis
+- Landmark detection
+
+This tutorial trains a custom image classification model to perform automated visual inspection of bridge decks to identify structures that are damaged by cracks.
+
+## ML.NET Image Classification API
+
+ML.NET provides various ways of performing image classification. This tutorial focuses on the Image Classification API. The Image Classification API makes use of [TensorFlow.NET](https://github.com/SciSharp/TensorFlow.NET), a low-level library that provides C# bindings for the TensorFlow C++ API.
+
+![](./media/image-classification-api-transfer-learning/architecture.png)
```

Comment:
Do you think an ML.NET user needs to know this level of detail?

Reply:
#resolved